roles/bigip_user/tasks/main.yaml

```diff
@@ -225,5 +265,5 @@
   register: result
 
 - name: Include version 13 (Daytona) checks
-  include: test-version-13-specific.yaml
+  include: test-post-version-12-specific.yaml
   when: system_info.product_information.product_version >= "13.0.0"
```

Comment:
you have to provide something in partition_access, in this case you can provide something like:
none:no-access. But this is mandatory otherwise API will complain, this is true for v12+, in v11, bigip filled in the blanks